### ISSUE TITLE
Why the performance of baseline and OGE are higher than that in the paper

### ISSUE THREAD

**Issue by shicaiwei123** [GeWu-Lab/OGM-GE_CVPR2022#34]
Hello,
Thanks for your inspiring work. 
However, I have some problems about the code. According to the provided pertained models, the performance on CREMA-D is 59.81 and 64.41, which is higher than 51.7 and 61.25 in the paper (Table 2).

What's the reason for these differences?

**Comment by echo0409** [comment]
Hi, thanks for your attention to our work.

We did some adjustment about the hyper-parameters (like learning rate). So the overall performance have a improvement compared to the results in the paper, but it does not affect the conclusion that our OGM keeps effective.
